eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -105,6 +113,20 @@ public function loadBinaryFileByUri($binaryFileUri)
     }
 
     /**
+     * @param $binaryFileUri
+     *
+     * @return MissingBinaryFile
+     */
+    private function createMissingBinaryFileByUri($binaryFileUri)
```

Comment:
I have a feeling that we don't need that. Creating a value object is a really simple operation, and the arguments speak for themselves.